streamalert_cli/terraform/generate.py

```diff
@@ -72,7 +72,6 @@ def _terraform_defaults(region):
         },
     })
 
-
```

Comment:
Shouldn't this newline be there? Two new lines between functions?

Reply:
he's correct - there should be 2 new lines here

Reply:
This was definitely unintentional. Thanks for the catch!